pkg/liqoctl/status/localinfo.go

```diff
@@ -100,14 +84,12 @@ func (lic *LocalInfoChecker) Collect(ctx context.Context) error {
 			networkSection.AddEntry("Reserved Subnets", ipamStorage.Spec.ReservedSubnets...)
 		}
 	}
-	apiServerAddress, err := util.ExtractValuesFromNestedMaps(values, "apiServer", "address")
+
+	apiServerAddress, err := apiserver.GetAddressFromMasterNode(ctx, lic.options.KubeClient)
```

Comment:
You should consider the case in which it is overridden through the arguments: https://github.com/liqotech/liqo/blob/95938948984c871d6a45e7819ee3be5277d9d186/pkg/utils/apiserver/address.go#L31-L40